src/repositories/raweventsRepository.ts

```diff
@@ -0,0 +1,28 @@
+let pgp = require('pg-promise');
+
+const connections = [];
+const dbName = process.env.DBPGNAME;
+const dbUser = process.env.DBPGUSER;
+const dbPassword = process.env.DBPGPASSWORD;
+const dbHost = process.env.DBPGHOST;
+const dbPort = process.env.DBPGPORT;
+const dbc = pgp({ capSQL: true});
+      console.log(`Opening connection to: ${dbName}, host is: ${dbHost}`);
+
+      const connectionString = `postgres://${dbUser}:${dbPassword}@${dbHost}:${dbPort}/${dbName}?ssl=true`;
+      connections[dbName] = dbc(connectionString);
+      let connection = connections[dbName];
+      
+export default class RawReportingEventsRepository {

```

Comment:
imo raw reporting event repository should take the 'connection' as an initialization parameter - so we can easily override it in unit tests

compare with 'repository.ts' file where model is passed this way